### ISSUE TITLE
Fatal error in HandleScope::HandleScope - Entering the V8 API without proper locking in place

### ISSUE THREAD

**Issue by NathanWalker** [NativeScript/ns-v8ios-runtime#85]
This issue can be experienced when using RadDataForm (nativescript-ui-dataform 7.0.4) in vanilla {N} apps using v8 runtime (7.0.x):

```
#
# Fatal error in HandleScope::HandleScope
# Entering the V8 API without proper locking in place
#
# From this line below: tns::Assert(obj->GetOwnPropertyNames(context).ToLocal(&properties), self->isolate_);
#
```

```
- (NSUInteger)count {
    Local<Object> obj = self->object_->Get(self->isolate_).As<Object>();

    if (obj->IsMap()) {
        return obj.As<Map>()->Size();
    }

    Local<Context> context = self->cache_->GetContext();
    Local<v8::Array> properties;
    tns::Assert(obj->GetOwnPropertyNames(context).ToLocal(&properties), self->isolate_);

    uint32_t length = properties->Length();

    return length;
}
```

Will try to create a small repro app.

**Comment by james-criscuolo** [comment]
I'm seeing this issue too. I'm running @nativescript/firebase off of a (now modified) pull request: https://github.com/EddyVerbruggen/nativescript-plugin-firebase/pull/1111

I know that sounds messy, but the code is running successfully in tns-ios 6.5.3. In xcode, the line that seems to trip up the code is here:

https://github.com/firebase/firebase-ios-sdk/blob/14764b8d60a6ad023d8fa5b7f81d42378d92e6fe/Firestore/Source/API/FSTUserDataConverter.mm#L199

The Dict that gets passed in seems to be fairly ordinary, so I'm not sure why it's failing here. I'm also a bit out of my element with this native code, so please let me know if I can provide something more helpful here.

**Comment by james-criscuolo** [comment]
This appears to be fixed in `7.1.0`, seems that #90 was what was needed.

**Comment by rigor789** [comment]
@james-criscuolo thanks for confirming!
